eprihlaska/views.py

```diff
@@ -830,6 +840,18 @@ def admin_set_state(id, state):
     return redirect(url_for('admin_list'))
 
 
+@app.route('/admin/payment_receipt/<id>')
+@require_remote_user
+def admin_payment_receipt(id):
+    application = ApplicationForm.query.filter_by(id=id).first()
```

Comment:
When the `id` is primary key, maybe a nicer query would be `ApplicationForm.query.get(id)`
http://flask-sqlalchemy.pocoo.org/2.3/queries/#querying-records

Reply:
Nice catch, thanks @Adman!